<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auth</class>
 <widget class="QWidget" name="Auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>156</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>156</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>156</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Authorization</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(29,33,38);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="topMargin">
    <number>7</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="exitbut">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 0px solid gray;
    border-radius: 7px;
    background-color: rgb(255,96,89);
	color:#ffffff;
}

QPushButton:hover {
	background-color: rgb(255,51,40);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minimize">
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 0px solid gray;
    border-radius: 7px;
    padding: 0 8px;
    background-color:rgb(255,189,46);
	color:#ffffff;
}

QPushButton:hover {
	background-color: rgb(255,168,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="back">
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 0px solid gray;
    border-radius: 7px;
    background-color:rgb(45,205,68);
	color:black;
}

QPushButton:hover {
	background-color:rgb(0,170,64);
}</string>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>Authorization</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Author">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <property name="verticalSpacing">
          <number>5</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="lable_login">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#a5a6a8;</string>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lable_pass">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#a5a6a8;</string>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="login">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border-radius: 8px;
	border-color: black;
    padding: 0 8px;
    background: rgb(46,53,61);
	color:rgb(255,255,255);
}</string>
           </property>
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="pass">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border-radius: 8px;
	border-color: black;
    padding: 0 8px;
    background: rgb(46,53,61);
	color:rgb(255,255,255);
}</string>
           </property>
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="sign_in">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 0px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background-color:rgb(0,199,77);
	color:#ffffff;
}

QPushButton:hover {
	background-color:rgb(0,170,64);
}</string>
           </property>
           <property name="text">
            <string>SIGN IN</string>
           </property>
           <property name="shortcut">
            <string>Return</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sign_up">
           <property name="minimumSize">
            <size>
             <width>127</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 0px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background-color:rgb(66,73,85);
	color:#ffffff;
}

QPushButton:hover {
	background-color:rgb(55,62,72);
}</string>
           </property>
           <property name="text">
            <string>SIGN UP</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Activate">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lable_login_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#a5a6a8;</string>
           </property>
           <property name="text">
            <string>AuthKey</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="auth_line">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border-radius: 8px;
	border-color: black;
    padding: 0 8px;
    background: rgb(46,53,61);
	color:rgb(255,255,255);
}</string>
           </property>
           <property name="maxLength">
            <number>19</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="activate_but">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 0px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background-color:rgb(0,199,77);
	color:#ffffff;
}

QPushButton:hover {
	background-color:rgb(0,170,64);
}</string>
           </property>
           <property name="text">
            <string>Activate</string>
           </property>
           <property name="shortcut">
            <string>Return</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cont_demo">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 0px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background-color:rgb(0,199,77);
	color:#ffffff;
}

QPushButton:hover {
	background-color:rgb(0,170,64);
}</string>
           </property>
           <property name="text">
            <string>Continue demo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Register">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lable_login_3">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#a5a6a8;</string>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="login_reg">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border-radius: 8px;
	border-color: black;
    padding: 0 8px;
    background: rgb(46,53,61);
	color:rgb(255,255,255);
}</string>
           </property>
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lable_pass_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#a5a6a8;</string>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="pass_reg">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border-radius: 8px;
	border-color: black;
    padding: 0 8px;
    background: rgb(46,53,61);
	color:rgb(255,255,255);
}</string>
           </property>
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lable_pass_3">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#a5a6a8;</string>
           </property>
           <property name="text">
            <string>E-mail</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="e_mail_reg">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border-radius: 8px;
	border-color: black;
    padding: 0 8px;
    background: rgb(46,53,61);
	color:rgb(255,255,255);
}</string>
           </property>
           <property name="maxLength">
            <number>64</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="register_button">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 0px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background-color:rgb(0,199,77);
	color:#ffffff;
}

QPushButton:hover {
	background-color:rgb(0,170,64);
}</string>
         </property>
         <property name="text">
          <string>SIGN UP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="status_db">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Сonnection...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
